src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int NUMBER_RANDOM = 100;

```

Comment:
```suggestion
    private static final int NUMBER_RANDOM = 101;
```
better use 101 because the value we pass to nextInt is excluded from possible values